xonsh/history/base.py

```diff
@@ -103,8 +103,8 @@ def append(self, cmd):
         Parameters
         ----------
         cmd: dict
-            A dict contains informations of a command. It should contains
-            the following keys like ``inp``, ``rtn``, ``ts`` etc.
+            A dict contains information on a command. It should contain
+            the following keys, like ``inp``, ``rtn``, ``ts`` &etc.
```

Comment:
I gave it another shot, using your rephrased sentence as a base, and further descriptions I drudged up from the `JsonHistory` and `HistoryEntry` classes. Since `JsonHistory` does not inherit the `append()` doctoring from `History`, I've updated it as well. Let me know if you'd still like to give it another round.